asab/web/container.py

```diff
@@ -159,6 +184,10 @@ async def _on_prepare_response(self, request, response):
 		if self.CORS == "*":
 			response.headers['Access-Control-Allow-Origin'] = "*"
 			response.headers['Access-Control-Allow-Methods'] = "GET, POST, DELETE, PUT, PATCH, OPTIONS"
+		elif self.CORS != "":
```

Comment:
This should be just `if self.CORS != ""` (no  elif) ... merge these two together